common/src/main/java/org/hawkular/accounts/common/ApplicationResources.java

```diff
@@ -85,6 +90,16 @@ public String getResourceNameSecret() {
         return secret;
     }
 
+    @Produces @HostSynonyms
+    public Set<String> getHostSynonyms() {
+        if (hostSynonyms == null) {
+            String synonyms = System.getProperty("org.hawkular.accounts.host.synonyms", "localhost,127.0.0.1,0.0.0.0");
```

Comment:
I'd keep this simple, as there's no way we can capture all the cases for this. I'd rather see a more complete solution at Keycloak's side, but I'm waiting for their answer on this topic.